guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,5 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
+Adam Ruzicka, Adi Abramovich, Amir Fefer, Amit Upadhye, Andrew Teixeira, Anna Vitova, Antoine Beaupré, Bastian Schmidt, Ben Magistro, Bernhard Suttner, Chris Roberts, Christopher Smith, Daniel Henninger, Eric D. Helms, Erik Berg, Evgeni Golov, Ewoud Kohl van Wijngaarden, Gordon Bleux, Ian Ballou, Jason, Jeremy Lenz, John Mitsch, Jonathon Turel, Justin Sherrill, Leos Stejskal, Lukáš Zapletal, Marcel Kühlhorn, Marek Hulán, Maria Agaphontzev, Markus Bucher, Melanie Corr, Nadja Heitmann, Nagoor Shaik, Oleh Fedorenko, Ondřej Ezr, Ondřej Pražák, Patrick Creech, Peter Koprda, Rahul Bajaj, Romuald Conty, Ron Lavi, Samir Jha, Sayan Das, Shimon Shtein, Shira Maximov, Tim Meusel, Tomer Brisker, William Clark, Yifat Makias
```

Comment:
There is a name "Jason" withtout any last name. Just wanted to point that out.

Reply:
I know, but I couldn't quite track it down.